src/sdk.ts

```diff
@@ -256,6 +256,72 @@ export class WarpSdk {
 
     return this.wallet.tx(txPayload);
   }
+
+  public async depositAccount(sender: string, account: string, token: Token, amount: string): Promise<TxInfo> {
+    let txPayload: CreateTxOptions;
+    if (token.type === 'cw20') {
+      txPayload = TxBuilder.new()
+        .execute<TransferMsg>(sender, token.token, {
+          transfer: {
+            amount,
+            recipient: account,
+          },
+        })
+        .build();
+    } else {
+      txPayload = TxBuilder.new()
+        .send(sender, account, { [token.denom]: amount })
+        .build();
+    }
+
+    return this.wallet.tx(txPayload);
+  }
+
+  public async withdrawAccount(owner: string, receiver: string, token: Token, amount: string): Promise<TxInfo> {
```

Comment:
all sdk tx functions should have first param set as `sender: string` which is the address of the tx signer - in this case owner == signer